react-graphql-client-library.md

```diff
@@ -65,7 +65,7 @@ const graphQLClient = axios.create();
 export default graphQLClient;
 {{< /highlight >}}
 
-Since you may want to have greater control when creating the GraphQL client instance, for example by passing in the GraphQL API endpoint or HTTP headers, you can expose it also with a function which returns the configured GraphQL client instance.
+Since you may need greater control for creating the GraphQL client instance, you can expose it also with a function that returns the configured GraphQL client instance.
```

Comment:
Why leaving out "for example by passing in the GraphQL API endpoint or HTTP headers"? It gives it more context, no?

Reply:
It didn't work with the commas.  It's an aside, meaning an interruption in the middle of a sentence, so it needs long dashes, like this:

Since you may need greater control for creating the GraphQL client instance--passing in the passing in the GraphQL API endpoint or HTTP headers, for example--you can aslo expose it with a function that returns the configured GraphQL client instance.

I changed it to what you see here. I am not sure if the long dashes translate in this HTML/markdown hybrid you're working with.